src/ia-bookmark-edit.js

```diff
@@ -1,37 +1,125 @@
-import { html, css, LitElement } from 'lit-element';
+import { nothing } from 'lit-html';
+import { repeat } from 'lit-html/directives/repeat.js';
+import { html, LitElement } from 'lit-element';
+import bookmarkEditCSS from './styles/ia-bookmark-edit.js';
 
 export class IABookmarkEdit extends LitElement {
   static get styles() {
-    return css`
-      :host {
-        display: block;
-        padding: 25px;
-        color: var(--ia-bookmarks-list-text-color, #000);
-      }
-    `;
+    return bookmarkEditCSS;
   }
 
   static get properties() {
     return {
-      title: { type: String },
-      counter: { type: Number },
+      bookmark: { type: Object },
+      renderHeader: { type: Boolean },
     };
   }
 
   constructor() {
     super();
-    this.title = 'Hey there';
-    this.counter = 5;
+    this.bookmark = {};
+    this.bookmarkColors = [{
+      id: 0,
+      className: '',
+    }, {
+      id: 1,
+      className: 'blue',
+    }, {
+      id: 2,
+      className: 'red',
+    }, {
+      id: 3,
+      className: 'green',
+    }, {
+      id: 4,
+      className: 'yellow',
+    }];
```

Comment:
QUESTION: Wondering if these "magic" numbers should be extracted to somewhere reusable. Does this mapping appear anywhere else?